ui/components/app/snaps/snap-ui-renderer/snap-ui-renderer.js

```diff
@@ -107,26 +110,29 @@ const SnapUIRendererComponent = ({
       </Box>
     </SnapDelineator>
   ) : (
-    <Box className="snap-ui-renderer__content">
+    <>
+      {header}
       <SnapInterfaceContextProvider
         snapId={snapId}
         interfaceId={interfaceId}
         initialState={initialState}
         context={context}
       >
-        <MetaMaskTemplateRenderer sections={sections} />
+        <Box className="snap-ui-renderer__content" height={BlockSize.Full}>
+          <MetaMaskTemplateRenderer sections={sections} />
+          {isPrompt && (
```

Comment:
This effectively means that the prompt input field is hidden when using `Container` and `Footer`

Reply:
Or actually all the time